tests/e2e/default/tasks/tasks.wdio-spec.js

```diff
@@ -41,8 +130,68 @@ describe('Tasks', () => {
     await browser.refresh();
   });
 
-  afterEach(async () => {
-    await utils.revertSettings(true);
+  it('should load multiple pages of tasks on infinite scrolling', async () => {
+    const settings = await compileTasks('tasks-multiple-config.js');
+    await utils.updateSettings(settings, { ignoreReload: 'api', sync: true });
+
+    await commonPage.goToTasks();
+    const list = await tasksPage.getTasks();
+    const infos = await tasksPage.getTasksListInfos(list);
+    expect(infos).to.have.length(201);
+    for (let i = 0; i < (infos.length/3); i++) {
```

Comment:
Since I have used 3 different contacts to create 67 tasks each I'd need to do this to validate each of the contact's task created using the [tasks-multiple-config.js](https://github.com/medic/cht-core/pull/9498/files#diff-9c786e98195c2ea9bcb9ba08ac6c090154cd10084a66ab5d75fe6b583d57296d) file

Reply:
Got it, thank you for the clarification